specification/ai/data-plane/HealthInsights/readme.md

```diff
@@ -4,10 +4,10 @@
 
 Configuration for generating Health Insights SDK.
 
-The current release is `2023-09-01-preview`.
+The current release is `2024-04-01`.
 
 ```yaml
-tag: 2023-09-01-preview
+tag: 2024-04-01
```

Comment:
Tag name in date format is causing failure in specs pipeline since it gets auto converted to long format date object and that does not match with any files.

Tag should be changed as `package-2024-04-01`